taskt/Core/Automation/Commands/Folder Operation Commands/CreateFolderCommand.cs

```diff
@@ -1,91 +1,90 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
 
     [Serializable]
-    [Attributes.ClassAttributes.Group("Folder Operation Commands")]
-    [Attributes.ClassAttributes.Description("This command creates a folder in a specified destination")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command to create a folder in a specific location.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements '' to achieve automation.")]
+    [Group("Folder Operation Commands")]
+    [Description("This command creates a folder in a specified location.")]
     public class CreateFolderCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the name of the new folder")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the name of the new folder.")]
-        [Attributes.PropertyAttributes.SampleUsage("myFolderName or [vFolderName]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("New Folder Name")]
+        [InputSpecification("Enter the name of the new folder.")]
+        [SampleUsage("myFolderName || {vFolderName}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_NewFolderName { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the directory for the new folder")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the directory.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\myfolder or [vTextFolderPath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Directory Path")]
+        [InputSpecification("Enter or Select the path to the directory to create folder in.")]
```

Comment:
change to "directory to create {a or the} folder in."

Reply:
done